packages/vaadin-grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -269,19 +271,9 @@ class GridProEditColumnElement extends GridColumnElement {
    * @protected
    */
   _stopCellEdit(cell, model) {
-    const editor = this._getEditorComponent(cell);
-    let shouldResize = true;
-    if (editor) {
-      editor.removeEventListener('focusout', this._grid.__boundEditorFocusOut);
-      editor.removeEventListener('focusin', this._grid.__boundEditorFocusIn);
-      editor.removeEventListener('internal-tab', this._grid.__boundCancelCellSwitch);
-    } else {
-      // avoid notify resize of editor removed due to scroll
-      shouldResize = false;
-    }
     document.body.removeEventListener('focusin', this._grid.__boundGlobalFocusIn);
+
     this._removeEditor(cell, model);
-    shouldResize && this._grid.notifyResize();
```

Comment:
Decided to remove the `notifyResize()` call since `notifyResize()` is [an empty method](https://github.com/vaadin/web-components/blob/9e3496ba20f3aab2b040790bbc496313289579d5/packages/vaadin-grid/src/vaadin-grid.js#L1015-L1017) and supposed to be removed in the future.